test/eventlistener_scale_test.go

```diff
@@ -32,16 +36,20 @@ import (
  */
 func TestEventListenerScale(t *testing.T) {
 	c, namespace := setup(t)
-	t.Parallel()
```

Comment:
@dibyom why do we want to remove `t.Parallel()` here?

Reply:
Ahh, did not intend for that to be in the change...but I found removing it made it easier to debug when running all e2e tests...running it in parallel mixes up the logs making it harder to track. And neither of the tests take too long so I'm inclined towards just removing the `t.Parallel()` in all of our e2e tests. What do you think?